usbreset.c

```diff
@@ -103,14 +108,19 @@ static void list_devices(void)
 	if (!devs)
 		return;
 
-	while ((dev = parse_devlist(devs)) != NULL)
-		printf("  Number %03d/%03d  ID %04x:%04x  %s\n", dev->bus_num, dev->dev_num, dev->vendor_id,
+	while ((dev = parse_devlist(devs)) != NULL) {
+		printf("  Number %03d/%03d  ID %04x:%04x  %s", dev->bus_num, dev->dev_num, dev->vendor_id,
 		       dev->product_id, dev->product_name);
+		if (strlen(dev->serial) > 0) {
```

Comment:
nit, the {} is not needed for one line if statements like this, but don't worry about it :)